src/assets/scss/modules/_dropdown.scss

```
@@ -38,4 +40,89 @@
       }
     }
   }
+
+  .dp-button-editor-wrap {
+    position: relative;
+    margin: variables.$dp-spaces--lv0;
+
+    .dp-button-box {
+      position: relative;
+    }
+
+    .dp-content-menu {
+      position: absolute;
+      display: none;
+      top: 40px;
+      text-align: left;
+      line-height: variables.$dp-spaces--lv4;
+      overflow: hidden;
+      left: 0;
+      outline: 0;
+      padding: 5px;
+      border-radius: 5px;
+      background-color: colors.$dp-color-white;
+      box-shadow: inset 0 0 0 3px colors.$dp-color-snow,
+        inset 0 0 0 5px colors.$dp-color-orange-focus;
+      z-index: 3;
+    }
+
+    .dp-list-exit {
+      margin: variables.$dp-spaces--lv0;
+      padding: variables.$dp-spaces--lv0;
+      overflow: auto;
+      width: 200px;
+      height: 150px;
+
+      /* Tamaño del scroll */
+      &::-webkit-scrollbar {
+        width: 4px;
+        scrollbar-width: 4px;
+      }
+
+      /* barra thumb de scroll */
+      &::-webkit-scrollbar-thumb {
```

Porque no usar la forma de construir de SASS?
```suggestion
      &::-webkit-scrollbar {
        &-thumb{
          background: colors.$dp-color-silver;
          border-radius: 4px;
          right: 10px;
        }
      }
      
```

---

Considero que se puede agrupar todo el comportamiento de ::-webkit-scrollbar en un solo bloque de código. Lo otro que pienso es que si vamos a escribir como CSS classic lo mejor seria cambiar la extensión del archivo a .css y así no dejamos la oportunidad de escribir CSS SASS con CSS classic